bin/node-sass

```diff
@@ -204,6 +246,15 @@ function run(options, emitter) {
     options.includePath = [options.includePath];
   }
 
+  if(isDirectory(options.src)) {
+    if (!options.output) {
+      emitter.emit('error', 'Specify an output directory when using multi-file compilation');
+    }
+    if (!isDirectory(options.output)) {
+      emitter.emit('error', ['Multi-file compilation: requires destination', options.output, 'to be a directory.'].join(' '));
```

Comment:
Also, I am not sure what the standard in `node-sass` is, but I prefer to use `util.format` for messages like this.   It is what node core does: https://github.com/joyent/node/blob/ef4344311e19a4f73c031508252b21712b22fe8a/lib/tls.js#L152